test-cases/PR-provision-test.yaml

```diff
@@ -23,3 +23,7 @@ gce_image_db: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/globa
 use_mgmt: true
 
 scylla_version: 3.2.rc1
+
+post_behavior_db_nodes: "keep-on-failure"
```

Comment:
I don't like doing this as default for provision tests

Reply:
why? if something fails we need to understand what the failure is and in most cases we need the instances. From my experience when we destroyed by default and had an issue we reran with keep to understand the root cause.

Reply:
Agree, because of the the "abuse" part